staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/dispatcher.go

```diff
@@ -121,17 +151,11 @@ func (a *mutatingDispatcher) Dispatch(ctx context.Context, attr admission.Attrib
 			continue
 		}
 
-		if versionedAttr == nil {
-			// First webhook, create versioned attributes
-			var err error
-			if versionedAttr, err = admission.NewVersionedAttributes(attr, invocation.Kind, o); err != nil {
-				return apierrors.NewInternalError(err)
-			}
-		} else {
-			// Subsequent webhook, convert existing versioned attributes to this webhook's version
-			if err := admission.ConvertVersionedAttributes(versionedAttr, invocation.Kind, o); err != nil {
-				return apierrors.NewInternalError(err)
-			}
+		// Subsequent webhook, convert existing versioned attributes to this webhook's version
+		var err error
+		versionedAttr, err = v.VersionedAttribute(invocation.Kind)
```

Comment:
can make this local to this iteration of the loop:

```suggestion
		versionedAttr, err := v.VersionedAttribute(invocation.Kind)
```

Reply:
yes